tests/core_tests.py

```diff
@@ -730,6 +747,7 @@ def test_fetch_datasource_metadata(self):
         for k in keys:
             self.assertIn(k, resp.keys())
 
+    @pytest.mark.usefixtures("load_birth_names_dashboard_with_slices")
```

Comment:
If this is replicated on almost every method, can we use the fixture at the module level?

Reply:
I think it' better to load fresh data for each test because some tests may modify data a bit. Using module fixture may result in creating some flaky tests.

Reply:
True, will probably load fast enough also